upi/vsphere/README.md

```diff
@@ -30,30 +31,29 @@ sshKey: YOUR_SSH_KEY
 
 3. Fill out a terraform.tfvars file with the ignition configs generated.
 There is an example terraform.tfvars file in this directory named terraform.tfvars.example. The example file is set up for use with the dev cluster running at vcsa.vmware.devcluster.openshift.com. At a minimum, you need to set values for the following variables.
+
 * cluster_id
 * cluster_domain
 * vsphere_user
 * vsphere_password
 * ipam_token
-* bootstrap_ignition_url
+* bootstrap_ignition_url => The bootstrap ignition config must be placed in a location that will be accessible by the bootstrap machine. For example, you could store the bootstrap ignition config in a gist.
 * control_plane_ignition
 * compute_ignition
-The bootstrap ignition config must be placed in a location that will be accessible by the bootstrap machine. For example, you could store the bootstrap ignition config in a gist.
 
-4. Run `terraform init`.
 
-5. Ensure that you have you AWS profile set and a region specified. The installation will use create AWS route53 resources for routing to the OpenShift cluster.
```

Comment:
Route53 is still being used for DNS, isn't it?

Reply:
that is correct @vrutkovs is still used hence why i've moved it up in the pre-req section so folks know upfront what is required from the existing code base.

I can extend if y'all okay with to mention why R53 provider is used and not DNS provider, let me know